network-mock/src/lib.rs

```diff
@@ -31,7 +31,7 @@ pub struct MockAddress(u64);
     From,
     Into,
 )]
-pub struct MockPeerId(u64);
+pub struct MockPeerId(pub u64);
```

Comment:
The `pub` seems unnecessary in this commit.
```suggestion
pub struct MockPeerId(u64);
```

Reply:
The test uses it to create a specific Peer to send a message to. I think without it being pub it cannot be used like so:
https://github.com/nimiq/core-rs-albatross/pull/2693/files#diff-026455fd64af083e01fb8f83c0d5430cd7e37ba6096fa32800f3a08a022f2deeR192

Reply:
Hmmm. That test kinda confuses `MockPeerId`s and handel's IDs. There's probably a mapping between them somewhere, no?

Reply:
Ah, it manually sets up the peer IDs for that to work.

You can go from `u64` to `MockPeerId` using `MockPeerId::from`.